stable/consul/templates/consul.yaml

```diff
@@ -37,6 +37,38 @@ spec:
       component: "{{ .Release.Name }}-{{ .Values.Component }}"
 {{- end }}
 ---
+{{- if .Values.uiIngress.enabled -}}
+{{- $releaseName := .Release.Name -}}
+{{- $servicePort := .Values.HttpPort -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
```

Comment:
This `Ingress` template lgtm.

I'm used to seeing these templates as separate files, but I see you're following the pattern existing in this chart, and idk if there's a set standard for that across charts. So this is 👍 as well - I'll follow up on this question separately later.